resources/compute-instance.go

```diff
@@ -0,0 +1,49 @@
+package resources
+
+// PricingInfo structure for storing the information from the billing API
```

Comment:
See https://golang.org/doc/effective_go.html#commentary. Comments documenting declarations should be full sentences, even if that seems a little redundant. This approach makes them format well when extracted into godoc documentation. Comments should begin with the name of the thing being described and end in a period: